modules/040-node-manager/templates/bashible/rbac-for-us.yaml

```diff
@@ -52,6 +52,20 @@ rules:
   verbs:
   - get
 
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: bashible-events
+  namespace: default
+  {{- include "helm_lib_module_labels" (list . ) | nindent 2 }}
+rules:
+  - apiGroups: ["events.k8s.io"]
+    resources:
+      - events
+    verbs:
+      - "*"
```

Comment:
Too broad, consider specifying only required verbs

Reply:
changed to get, list, create, update, patch